.github/workflows/asan.yml

```
@@ -0,0 +1,187 @@
+name: Sanitizers
+on:
+  push:
+    branches: [ master ]
```

YAMLlint (1.38.0)

[error] too many spaces inside brackets

(brackets)